src/main/java/dk/cs/aau/huppaal/presentations/HUPPAALPresentation.java

```diff
@@ -331,20 +318,154 @@ private void initializeColorSelector() {
             controller.generateUppaalModel.setEnabled(true);
-            controller.generateUppaalModel.setOpacity(1);
+            controller.generateUppaalModel.setOpacity(1.0);
         }
     }
 
+    private void initializeRunConfigExecuteButton() {
+        var colorIntensity = Color.Intensity.I800;
+        var color = Color.GREY_BLUE;
+        controller.runConfigurationExecuteButton.setRipplerFill(color.getTextColor(colorIntensity));
+
+        JFXTooltip tooltip;
+        var c = controller.runConfigurationPicker.getSelectionModel().getSelectedItem();
+        if(c == null || c.runConfiguration().isEmpty()) {
+            tooltip = new JFXTooltip("no run configuration is selected");
+            controller.runConfigurationExecuteButton.setEnabled(false);
+            controller.runConfigurationExecuteButton.setOpacity(0.3);
+            controller.runConfigurationExecuteButton.setOnMouseClicked(e -> HUPPAAL.showToast("no run configuration is selected"));
+        } else {
+            tooltip = new JFXTooltip("Run " + c.runConfiguration().get().name);
+            controller.runConfigurationExecuteButton.setEnabled(true);
+            controller.runConfigurationExecuteButton.setOpacity(1.0);
+            controller.runConfigurationExecuteButton.setOnMouseClicked(e -> executeRunConfiguration(c.runConfiguration().get()));
+        }
+        JFXTooltip.install(controller.runConfigurationExecuteButton, tooltip);
+    }
+
+    private final static String[] sysEnv = System.getenv().entrySet().stream().map((e) -> e.getKey() + "=" + e.getValue()).toArray(String[]::new);
+    private Process proc;
+    private void executeRunConfiguration(RunConfiguration config) {
+        // Stop the currently running process if it is running
+        if(proc != null && proc.isAlive()) {
+            proc.destroy();
+            return;
+        }
+
+        // Else start the run configuration
+        new Thread(() -> {
+            try {
+                var rt = Runtime.getRuntime();
+                if(config.program.isEmpty())
+                    throw new Exception("No program to run in selected run configuration");
+                var dir = new File(config.executionDir);
+                if(!(dir.exists() && dir.isDirectory()))
+                    throw new Exception(String.format("'%s' does not exist or is not a directory", config.executionDir));
+                proc = rt.exec(config.program + " " + config.arguments,
+                                ArrayUtils.merge(sysEnv, config.environmentVariables.split(";")),
+                                dir);
+                var stdi = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+                var stde = new BufferedReader(new InputStreamReader(proc.getErrorStream()));
+                controller.runConfigurationExecuteButtonIcon.setIconLiteral("gmi-stop");
+                controller.runConfigurationExecuteButtonIcon.setIconColor(javafx.scene.paint.Color.web("#ff7e79"));
```

Comment:
Is there a reason not to use one of the colors specified in [colors.css](https://github.com/DEIS-Tools/H-Uppaal/blob/master/src/main/resources/colors.css) or one of the predefined colors in `javafx.scene.paint.Color`?

Reply:
My bad, I guess I forgot that they existed at the time 😄 

Fixed in a5ebb85ebe763dff085dccb674847ffba26269eb